Tools/autotest/models/freestyle.param

```diff
@@ -10,34 +10,11 @@ ARMING_RUDDER    0.000000
 ATC_ACCEL_P_MAX  160000.000000
 ATC_ACCEL_R_MAX  160000.000000
 ATC_ACCEL_Y_MAX  120000.000000
-ATC_ANG_PIT_P    15.000000
-ATC_ANG_RLL_P    15.000000
-ATC_ANG_YAW_P    10.000000
 ATC_INPUT_TC     0.040000
-ATC_RAT_PIT_P    0.040000
-ATC_RAT_PIT_I    0.040000
-ATC_RAT_PIT_D    0.001200
-ATC_RAT_PIT_FLTD 60.000000
-ATC_RAT_PIT_FLTE 40.000000
```

Comment:
I'm confused as to why this tuning is being removed.  I thought this was in here as a general testing vehicle, not just something to run autotune against?!

Reply:
@MichelleRos can you comment on this? should I back out these changes?